docs/classes/response.md

```diff
@@ -3,92 +3,113 @@ title: Response
 permalink: /classes/response
 ---
 
-{:.warning}
-> This page needs an update.
-
 * Table of Contents
 {:toc}
 
-The `Response` interface (extends `\Amp\ByteStream\OutputStream`) generally finds its only use in responder callables (or [`Websocket::onOpen()`](websocket.md#onopenint-clientid-handshakedata)). [`Middleware`s](middleware.md) do never see the `Response`; the `StandardResponse` class is communicating headers, data and flushes to a Generator under the hood.
+The **`Response`** class represents an HTTP response. The **`Response`** promise is returned by [request handlers](request-handler.md) and [`middleware`](middleware.md).
```

Comment:
`middleware` shouldn't have code tags here or should be `Middleware`, but it should be consistent with `request handlers`.